File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int zeroCount=0;
        int oneCount=0;
        int n=nums.length;
        for(int i:nums){
            if (i==0)
                zeroCount++;
            else if(i==1)
                oneCount++;
            
        }
        for( int j=0;j< n ; j++){
            if ( j < zeroCount)
                nums[j]=0;
            else if ( j < zeroCount + oneCount)
                nums[j]=1;
            else
                nums[j]=2;

        }
        
        
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        i,j,k=(0,0,len(nums)-1)
        while (j <= k):
            if nums[j]==0:
                nums[j],nums[i]=nums[i],nums[j]
                j+=1
                i+=1
            elif nums[j]==2:
                nums[k],nums[j]=nums[j],nums[k]
                k-=1
                
            else:
                j+=1


        